protocols/smtp/src/test/java/org/apache/james/protocols/smtp/utils/BaseFakeSMTPSession.java

```diff
@@ -27,6 +27,7 @@
 
 import org.apache.james.core.Username;
 import org.apache.james.protocols.api.ProtocolSession;
+import org.apache.james.protocols.api.ProxyInformation;
```

Comment:
Are we missing the ProxyInformation file in this pull request?

Reply:
Resolved.